train.py

```diff
@@ -115,13 +115,16 @@ def train(args, log_dir, writer, logger):
         model = get_model(args.arch, n_classes)
         criterion = UncertaintyLoss(input_slice=input_slice)
 
-    model.cuda()
-
     # Drawing graph for TensorBoard
     dummy = torch.zeros((2, 3, args.image_size, args.image_size)).cuda()
     model(dummy)
     writer.add_graph(model, dummy)
 
+    if torch.cuda.device_count() > 1:
```

Comment:
Add an cli option to enable as well?

Reply:
Changed it so it runs in parallel if --gpu is not given.